recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-graphics/images/core-image-weston.bbappend

```diff
@@ -0,0 +1,27 @@
+IMAGE_INSTALL_append = " \
+    pulseaudio \
+    alsa-utils \
+    kernel-modules \
+    kmscube \
+"
+
+# Do not support secure environment
+IMAGE_INSTALL_remove = " \
+    optee-linuxdriver \
+    optee-linuxdriver-armtz \
+    optee-client \
+    libx11-locale \
```

Comment:
libx11-locale has nothing to do with optee

Reply:
done